pkg/build/strategies/layered/layered.go

```diff
@@ -150,18 +151,20 @@ func (builder *Layered) Build(config *api.Config) (*api.Result, error) {
 		buildResult.BuildInfo.FailureReason = utilstatus.NewFailureReason(utilstatus.ReasonGenericS2IBuildFailed, utilstatus.ReasonMessageGenericS2iBuildFailed)
 		return buildResult, errors.New("builder image name cannot be empty")
 	}
-
+	timeStart = time.Now()
 	if err := builder.CreateDockerfile(config); err != nil {
```

Comment:
p.s.: just for the record, i do see a value in that, but is it worth the code duplication?